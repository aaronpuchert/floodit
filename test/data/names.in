2 2
0 0
red blue
blue red
