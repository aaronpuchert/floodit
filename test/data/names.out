A shortest sequence of 2 moves is given by:

    [red] blue red
